alpine linux llgo run demo panic

1、os
alpine linux 3.21

2、llgo version
`llgo v0.9.8 linux/amd64`

3、code
package main

import "github.com/goplus/llgo/c"

func main() {
	c.Printf(c.Str("Hello world\n"))
}

4、llgo run .
panic: test link args ' $(pkg-config --libs bdw-gc); -lgc' failed
		expanded to: [-lgc -lpthread -ldl]
		resolved to: [-lgc -lpthread -ldl]
		error: 

goroutine 1 [running]:
github.com/goplus/llgo/internal/build.buildAllPkgs(0xc00887cf60, {0xc003e1eb40?, 0xc006f5fc80?, 0x8ed3e0?}, 0x0)
	/go/pkg/mod/github.com/goplus/llgo@v0.9.8/internal/build/build.go:344 +0xbec
github.com/goplus/llgo/internal/build.Do({0xc000024470?, 0x1?, 0x1?}, 0xc00007e5a0)
	/go/pkg/mod/github.com/goplus/llgo@v0.9.8/internal/build/build.go:214 +0xc1e
github.com/goplus/llgo/cmd/internal/run.runCmdEx(0xc000065e58?, {0xc000024470, 0x1, 0x1}, 0x2)
	/go/pkg/mod/github.com/goplus/llgo@v0.9.8/cmd/internal/run/run.go:68 +0x105
github.com/goplus/llgo/cmd/internal/run.runCmd(0xc0000ae000?, {0xc000024470?, 0x0?, 0x0?})
	/go/pkg/mod/github.com/goplus/llgo@v0.9.8/cmd/internal/run/run.go:52 +0x1d
main.main()
	/go/pkg/mod/github.com/goplus/llgo@v0.9.8/cmd/llgo/llgo.go:90 +0x4b3


llgo compiled in alpine linux3.21 docker (go version 1.23.3)，because llgo does not support alpine linux?

--- a/README.md
+++ b/README.md
@@ -341,8 +341,8 @@ Here are the Go packages that can be imported correctly:
 
 - [Go 1.20+](https://go.dev)
 - [LLVM 18](https://llvm.org)
-- [LLD 18](https://lld.llvm.org)
 - [Clang 18](https://clang.llvm.org)
+- [LLD 18](https://lld.llvm.org)
 - [pkg-config 0.29+](https://www.freedesktop.org/wiki/Software/pkg-config/)
 - [bdwgc/libgc 8.0+](https://www.hboehm.info/gc/)
 - [OpenSSL 3.0+](https://www.openssl.org/)
@@ -366,7 +366,9 @@ go install -v github.com/goplus/llgo/cmd/llgo@latest
 
 ```
 
-### on Linux (Debian/Ubuntu)
+### on Linux
+
+#### Debian/Ubuntu
 
 <!-- embedme doc/_readme/scripts/install_ubuntu.sh#L2-L1000 -->
 
@@ -377,6 +379,19 @@ sudo apt-get update
 sudo apt-get install -y llvm-18-dev clang-18 libclang-18-dev lld-18 pkg-config libgc-dev libssl-dev zlib1g-dev libcjson-dev libsqlite3-dev
 sudo apt-get install -y python3.12-dev # optional
 go install -v github.com/goplus/llgo/cmd/llgo@latest
+```
+
+#### Alpine Linux
+
+```sh
+apk add go llvm18-dev clang18-dev lld18 pkgconf gc-dev openssl-dev zlib-dev
+apk add python3-dev # optional
+apk add g++ # build only
+export LLVM_CONFIG=/usr/lib/llvm18/bin/llvm-config
+export CGO_CPPFLAGS="$($LLVM_CONFIG --cppflags)"
+export CGO_CXXFLAGS=-std=c++17
+export CGO_LDFLAGS="$($LLVM_CONFIG --ldflags) $($LLVM_CONFIG --libs all)"
+go install -v -tags=byollvm -ldflags="-X github.com/goplus/llgo/xtool/env/llvm.ldLLVMConfigBin=$LLVM_CONFIG" github.com/goplus/llgo/cmd/llgo@latest
 ```
 
 ### on Windows

